pkgs/misc/qtscrcpy/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libsForQt5
+, copyDesktopItems
+, android-tools
+, makeDesktopItem
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qtscrcpy";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "barry-ran";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-2d7AIUra7Uc/N0Ako8JYo07GUTPNgorxl2hmeDJOGPU=";
+  };
+
+  nativeBuildInputs = [ cmake libsForQt5.wrapQtAppsHook copyDesktopItems ];
+  buildInputs = with libsForQt5.qt5 ;[ qttools.dev qtx11extras.dev ];
+
+  patchPhase = ''
+    substituteInPlace ./QtScrcpy/main.cpp --replace "../../../third_party/adb/linux/adb" "${android-tools.out}/bin/adb"
+    substituteInPlace ./QtScrcpy/main.cpp --replace "../../../third_party/scrcpy-server" "$out/scrcpy-server"
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin/
+    pwd
```

Comment:
Drop this perhaps?

Reply:
`pwd` for debugging and forget to delete...